EnglishFilter/sections/foreign.txt

```diff
@@ -2996,6 +2996,8 @@ kava.ee##.kava_ad_chilli
 !------------------------------ Finnish ------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/78743
+tennisassa.fi#$?#.page-content__main > div.module > div.module__content div.ad--container:upward(div.module) { remove: true; }
```

Comment:
Please re-write this one a bit
`tennisassa.fi#$?#.page-content__main > div.module:first-child > div.module__content div.ad--container:upward(div.module) { remove: true; }`